static/src/javascripts/projects/common/modules/experiments/affix.js

```diff
@@ -2,19 +2,23 @@ define([
     'bean',
     'bonzo',
     'common/utils/_',
-    'common/utils/mediator'
+    'common/utils/mediator',
+    'fastdom'
 ], function (
     bean,
     bonzo,
     _,
-    mediator
+    mediator,
+    fastdom
 ) {
 
     var Affix = function (options) {
 
-        bean.on(window, 'click', this.checkPositionWithEventLoop.bind(this));
-        mediator.addListener('window:scroll', _.debounce(this.checkPositionWithEventLoop.bind(this), 10));
-        mediator.addListener('window:resize', _.debounce(this.calculateContainerPositioning.bind(this), 200));
+        _.bindAll(this, 'checkPosition', 'calculateContainerPositioning');
+
+        bean.on(window, 'click', this.checkPosition);
+        mediator.addListener('window:throttledScroll', _.debounce(this.checkPosition, 10));
```

Comment:
Throttling by `10` is probably useless. I'd rather make it `50`, or remove it completely